src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ReviewTimeExtensionsHandler.java

```diff
@@ -41,31 +56,85 @@ public boolean canHandle(
             throw new IllegalStateException("Cannot handle callback");
         }
 
+        //TODO Only return date is extension is granted
         AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
```

Comment:
Do you not need to do the todo?

Reply:
This will be blank won't it if the time extension is rejected then can't be parsed